cli/web_worker.rs

```diff
@@ -23,17 +65,48 @@ use std::task::Poll;
 /// `WebWorker`.
 pub struct WebWorker {
   worker: Worker,
-  is_ready: bool,
+  event_loop_idle: bool,
+
+  pub terminated: Arc<AtomicBool>,
```

Comment:
Is this shared with `WebWorkerHandle`? Wouldn't it be better to just have a reference to the `WebWorkerHandle` ?